Dao/Configuration.php

```diff
@@ -85,8 +86,11 @@ public function getCustomDimension($idDimension, $idSite)
 
     public function getCustomDimensionsHavingScope($idSite, $scope)
     {
-        $query= "SELECT * FROM " . $this->tableNamePrefixed . " WHERE idsite = ? and scope = ?";
-        return $this->fetchAllDimensionsEnriched($query, array($idSite, $scope));
+        $result = Request::processRequest('CustomDimensions.getConfiguredCustomDimensions', [
```

Comment:
the DAO should it theory only include DB queries. Should we instead move the `getCustomDimensionsHavingScope ` method to the API so consumers can call this method from there? like `CustomDimensions. getCustomDimensionsHavingScope `?

Reply:
I can do that. Or I can add it to another class to avoid another supported API method.

Reply:
just put it in the api